custom_addons/softifi_graph_module_dependency/models/ir_model.py

```
@@ -0,0 +1,91 @@
+from odoo import models, api
+from .graph_builder import GraphBuilderMixin
+
+
+class IrModel(models.Model):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)